tests/typecheck/utils/test_functional.yml

```diff
@@ -8,7 +8,7 @@
           def attr(self) -> List[str]: ...
 
           reveal_type(attr)      # N: Revealed type is "django.utils.functional.cached_property[builtins.list[builtins.str]]"
-          reveal_type(attr.name) # N: Revealed type is "builtins.str"
+          reveal_type(attr.name) # N: Revealed type is "Union[builtins.str, None]"
```

Comment:
Implementation indicates:
"The name argument is deprecated as it's unnecessary as of Python 3.6.",
  RemovedInDjango50Warning,